lib/react_client.dart

```diff
@@ -36,19 +36,100 @@ newJsMap(Map map) {
 }
 
 /**
- * Type of [children] must be child or list of childs, when child is JsObject or String
+ * Type of [children] must be child or list of children, when child is JsObject or String
  */
 typedef JsObject ReactComponentFactory(Map props, [dynamic children]);
 typedef Component ComponentFactory();
 
 class ReactComponentFactoryProxy implements Function {
-  final ReactComponentFactory _call;
   final JsFunction reactComponentFactory;
-  ReactComponentFactoryProxy(this.reactComponentFactory, this._call);
+  ReactComponentFactoryProxy(this.reactComponentFactory);
 
   JsObject call(Map props, [dynamic children]) {
-    return this._call(props, children);
+
+    if (children == null) {
+      children = [];
+    } else if (children is! Iterable) {
+      children = [children];
```

Comment:
good call.  based on some other code that changed, this section isn't even necessary